<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>620</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tool-0.4.0</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="group_src">
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>原图</string>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>77</width>
        <height>112</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btnLoadImg">
         <property name="text">
          <string>加载</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveImg">
         <property name="text">
          <string>保存</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnCapture">
         <property name="text">
          <string>抓图</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEditImg">
         <property name="text">
          <string>编辑</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_src">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>231</width>
        <height>91</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>src rect...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="group_bin">
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>130</height>
      </size>
     </property>
     <property name="title">
      <string>二值化</string>
     </property>
     <widget class="QLabel" name="label_bin">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>211</width>
        <height>91</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>2</width>
        <height>2</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>200</height>
       </size>
      </property>
      <property name="text">
       <string>binary rect...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>提取点阵</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>整体提取</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="2" rowspan="3">
    <widget class="QGroupBox" name="group_dict">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>340</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>字库</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="3">
       <widget class="QPushButton" name="btnnewdict">
        <property name="text">
         <string>新建字库</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3" colspan="3">
       <widget class="QPushButton" name="pushButton_7">
        <property name="text">
         <string>加载字库</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QPushButton" name="pushButton_9">
        <property name="text">
         <string>编辑/显示字库</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3" colspan="3">
       <widget class="QPushButton" name="pushButton_8">
        <property name="text">
         <string>保存字库</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>路径:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="5">
       <widget class="QLineEdit" name="editPath">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Dict Info:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="4" colspan="2">
       <widget class="QPushButton" name="pushButton_11">
        <property name="text">
         <string>系统字库</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0" colspan="6">
       <widget class="QListView" name="listView"/>
      </item>
      <item row="5" column="0" colspan="4">
       <widget class="QLabel" name="label">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>定义文字(回车添加到字库）：</string>
        </property>
       </widget>
      </item>
      <item row="5" column="4" colspan="2">
       <widget class="QLineEdit" name="lineEdit">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>150</height>
         </size>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="2">
       <widget class="QLineEdit" name="editFind"/>
      </item>
      <item row="6" column="2" colspan="3">
       <widget class="QPushButton" name="btnFind">
        <property name="text">
         <string>查找</string>
        </property>
       </widget>
      </item>
      <item row="6" column="5">
       <widget class="QPushButton" name="pushButton_6">
        <property name="text">
         <string>删除选中</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" rowspan="2">
    <widget class="QGroupBox" name="group_info">
     <property name="title">
      <string>颜色信息</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableView" name="tableView">
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QCheckBox" name="chkBk">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>背景</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_3"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="group_parray">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>320</height>
      </size>
     </property>
     <property name="title">
      <string>点阵</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>334</width>
           <height>344</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="group_ocr">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>80</height>
      </size>
     </property>
     <property name="title">
      <string>ocr测试</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>sim</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="doubleSpinBoxSim">
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="maximumSize">
         <size>
          <width>1000</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
